app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,7 @@
 class PostsController < ApplicationController
   def index
-    @posts = Post.all
-    @user = User.includes(posts: [:comments]).find(params[:user_id].to_i)
+    @user = User.find(params[:user_id].to_i)
+    @posts = @user.posts.includes([:author])
   end
```

Comment:
- [ ] To fix `n+1` problem, Please make line 4 like this:
```ruby
@user.posts.includes(:comments)
```

Reply:
Thank you for your review but this suggestion is creating a bullet error for me.
Currently I don't have any N+1 error  

<img width="488" alt="image" src="https://user-images.githubusercontent.com/106079814/200935580-2d981657-f57b-49f7-ae82-bf536d658842.png">